docs/secure-log-path-variable.md

```diff
@@ -14,6 +12,85 @@ Implemented in Percona Server for MySQL 8.0.28-19 (2022-05-12).
 | Data type     | String            |
 | Default       | empty string      |
 
-This variable restricts the dynamic log file locations. The variable is read-only and must be set up in a configuration file or the command line.
+This variable restricts the dynamic log file locations. The variable is read-only and is set up in a configuration file or the command line.
+
+The accepted value is the directory name as a string. The default value is an empty string. When the value is an empty string, the variable only adds a warning to the error log and does nothing and the slow quary log and the general log are located in the default, /var/lib/mysql, directory. If the value contains a directory name, then the slow query log and the general log are located in that directory. An attempt to move either of these files outside of the specified directory results in an error.
+
+## The example of the secure_log_path variable usage
+
+1. Create the direcory to store the slow query log and the general log.
+
+    ```{.bash data-prompt="[root@localhost ~]#"}
```

Comment:
Why are we using 
root@localhost ~]#
It doesn't mention that the user must be root.